protocol/bc/block.go

```diff
@@ -106,7 +106,6 @@ func (b *Block) WriteTo(w io.Writer) (int64, error) {
 	return ew.Written(), ew.Err()
 }
 
-// assumes w has sticky errors
```

Comment:
This one doesn't check errors, so would assume it has sticky errors. We should either leave comment or check errors.